File: dp_labs_and_solutions/dp_labs_and_solutions/dp_cpp_labs_and_solutions/bridge/lab/rectangle.cpp
// rectangle.cpp

#include "rectangle.h"

rectangle::rectangle(double a_height, double a_width)
{
	set_height(a_height);
	set_width (a_width );
}

double rectangle::get_height() const
{
	return my_height;
}

double rectangle::get_width() const
{
	return my_width;
}

void rectangle::set_height(double a_height)
{
	my_height = a_height;
}

void rectangle::set_width(double a_width)
{
	my_width = a_width;
}

File: dp_labs_and_solutions/dp_labs_and_solutions/dp_java_labs_and_solutions/bridge/lab/Rectangle.java
// Rectangle.java

class Rectangle extends RightAngledQuadrilateral
{
	public Rectangle(double aHeight, double aWidth)
	{
		setHeight(aHeight);
		setWidth (aWidth );
	}

	public double getHeight()
	{
		return myHeight;
	}

	public double getWidth()
	{
		return myWidth;
	}

	public void setHeight(double aHeight)
	{
		myHeight = aHeight;
	}

	public void setWidth(double aWidth)
	{
		myWidth = aWidth;
	}

	private double myHeight;
	private double myWidth ;
}
